Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -57,13 +57,17 @@ class TopTabsViewController: UIViewController, Themeable, Notifiable {
         button.semanticContentAttribute = .forceLeftToRight
         button.addTarget(self, action: #selector(TopTabsViewController.tabsTrayTapped), for: .touchUpInside)
         button.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.tabsButton
+        button.showsLargeContentViewer = true
     }
 
     private lazy var newTab: UIButton = .build { button in
         button.setImage(UIImage.templateImageNamed(StandardImageIdentifiers.Large.plus), for: .normal)
         button.semanticContentAttribute = .forceLeftToRight
         button.addTarget(self, action: #selector(TopTabsViewController.newTabTapped), for: .touchUpInside)
         button.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.addNewTabButton
+        button.accessibilityLabel = .AddTabAccessibilityLabel
+        button.showsLargeContentViewer = true
+        button.largeContentTitle = .AddTabAccessibilityLabel
```

Comment:
Here I added a missing accessibility label with an existing accessibility label and that  accessibility label. Should I create a separate localized string for the `largeContentTitle`?